src/resources/styles-data.ts

```diff
@@ -154,4 +154,12 @@ export const derivedStyles = {
   "material-background-color": "var(--card-background-color)",
   "material-secondary-background-color": "var(--secondary-background-color)",
   "material-secondary-text-color": "var(--secondary-text-color)",
+  "safe-area-bottom":
```

Comment:
Lets not change the name
```suggestion
  "safe-area-inset-bottom":
```

Reply:
Done, thanks